temporary.py

```diff
@@ -37,17 +42,27 @@
 fl.next("ph,freq -- apply receiver phase")
 d /= phasing
 fl.image(d.C.setaxis("power", "#").set_units("power", "scan #"))
+fl.next("ph,freq -- transients in phase")
 ph0 = d.C.sum('t2')
 ph0 /= abs(ph0)
 d /= ph0
-fl.next("ph,freq -- transients in phase")
-fl.image(d.C.setaxis("power", "#").set_units("power", "scan #"))
+fl.image(d['t2':(-500,500)].C.setaxis("power", "#").set_units("power", "scan #"))
+fl.next(r'after correlation alignment, $\varphi$ domain')
 opt_shift,sigma = correl_align(d, indirect_dim='power',
-        ph1_selection=0,ph2_selection-0,sigma=50)
+        ph1_selection=0,ph2_selection=0,sigma=1e3)
 d.ift('t2')
 d *= np.exp(-1j*2*pi*opt_shift*d.fromaxis('t2'))
 d.ft('t2')
 fl.basename = None
-fl.next(r'after correlation alignment, $\varphi$ domain')
-fl.image(d)
-fl.show()
+fl.image(d['t2':(-500,500)].C.setaxis('power','#').set_units('power','scan #'))
+fl.next(r'after masked correlation alignment, $\varphi$ domain')
+opt_shift,sigma = correl_align(d, indirect_dim='power',
+        ph1_selection=0,ph2_selection=0,sigma=30e3/2.35)
```

Comment:
where did the choice of the sigma selection come from? I know the divided by 2.35 has to do with the linewidth and such but what is the 30e3?

Reply:
I realized this is a typo -- I mean it to be 30 Hz -- it's interesting that the result here is better I'm playing with it now.

Reply:
[see here](https://jmfrancklab.slack.com/archives/CLMMYDD98/p1622645958002900)

Reply:
by eye, that's what it looked like to me